docs/index.md

```diff
@@ -1,33 +1,54 @@
 
-In the dependency model, extensions can rely on each other for required services that are out of scope of the primary purpose of an extension, allowing each extension to focus on a specific purpose. 
+* get installed, potentially causing other dependency extensions to be installed as well
+* can be customized through configuration at runtime
+* get upgraded to newer version/s following upgrade paths defined by extension developers
+* and finally, get decommissioned and removed.
 
-OLM also prevents conflicting extensions from running on the cluster, either with conflicting dependency constraints or conflicts in ownership of services provided via APIs. Since cluster extensions need to be supported with an enterprise-grade product lifecycle, there has been a growing need for allowing extension authors to limit installation and upgrade of their extension by specifying additional environmental constraints as dependencies, primarily to align with what was tested by the extension author's QE processes. In other words, there is an evergrowing ask for OLM to allow the author to enforce these support limitations in the form of additional constraints specified by extension authors in their packaging for OLM.
+The first iteration of OLM, termed v0, included several concepts and features targeting the stability, security, and supportability of the life-cycled applications, for instance:
+* A dependency model that enabled cluster extensions to focus on their primary purpose by delegating out of scope behavior to dependencies.
+* A constraint model that allowed cluster extension developers to define support limitations such as conflicting extensions, and minimum kubernetes versions
+* A namespace-based multi-tenancy model in lieu of namespace-scoped CRDs.  
+* A packaging model in which catalogs of extensions, usually containing the entire version history of each extension, are made available to clusters for cluster users to browse and select from.
 
-During their lifecycle on the cluster, OLM also manages the permissions and capabilities extensions have on the cluster as well as the permission and access tenants on the cluster have to the extensions. This is done using the Kubernetes RBAC system, in combination with tenant isolation using Kubernetes namespaces. While the interaction surface of the extensions is solely composed of Kubernetes APIs the extensions define, there is an acute need to rethink the way tenant(i.e consumers of extensions) isolation is achieved. The ask from OLM, is to provide tenant isolation in a more intuitive way than [is implemented in OLM v0](https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/#docs)
+Since its initial release, OLM has helped catalyse the growth of operators throughout the Kubernetes ecosystem. [OperatorHub.io](https://operatorhub.io/)
+is a popular destination for discovering operators, and boasts over 300 packages from a many different vendors.
```

Comment:
```suggestion
Since its initial release, OLM has helped catalyse the growth of Operators throughout the Kubernetes ecosystem. [OperatorHub.io](https://operatorhub.io/)
is a popular destination for discovering Operators, and boasts over 300 packages from many different vendors.
```